core/includes/form.inc

```diff
@@ -4470,6 +4470,17 @@ function theme_file($variables) {
   return '<input' . backdrop_attributes($element['#attributes']) . ' />';
 }
 
+ /**
+ * Processes a file upload element, make use of #multiple if present.
+ */
+function form_process_file($element) {
+  if ($element['#multiple']) {
```

Comment:
According to https://api.backdropcms.org/form_api#multiple, `#multiple` applies only to `select`, and `tableselect`; not to files. Am I missing something?

Reply:
Yes, but this PR aims to change just that.